pkg/cli/alpha/config-gen/controller-gen-filter.go

```diff
@@ -64,7 +64,10 @@ func (cgr ControllerGenFilter) Filter(input []*yaml.RNode) ([]*yaml.RNode, error
 
 	// set the directory
 	b := bufferedGenerator{}
-	rt, _ := gens.ForRoots(cgr.Spec.CRDs.SourceDirectory) // ignore the spurious error
```

Comment:
it was ignoring the errors
Why do we need to return the error to bump 1.18?

Reply:
Found : https://github.com/kubernetes-sigs/kubebuilder/pull/2581#pullrequestreview-928178949 👍

Reply:
Could you add that it also solves : Closes; https://github.com/kubernetes-sigs/kubebuilder/issues/2567
Or indeed would be great if we could merge the fix and the bump in diff PRs